deployment/updates/scripts/xml/2018_01_30_UpdateServerNodeDownTemplate.template.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xml>
+    <config>
+        <serviceUrl>@SERVICE_URL@</serviceUrl>
+        <partnerId>-2</partnerId>
+        <clientTag>Stand alone php 1.0.0</clientTag>
+        <curlTimeout>30</curlTimeout>
+        <userAgent>Stand alone php 1.0.0</userAgent>
+    </config>
+    <session>
+        <partnerId>-2</partnerId>
+        <secret>@ADMIN_CONSOLE_PARTNER_ADMIN_SECRET@</secret>
+        <sessionType>2</sessionType>
+    </session>
+
+    <multirequest>
+        <!-- Server-Node is down -->
+        <request service="eventNotificationTemplate" action="listTemplates" plugin="eventNotification">
+            <filter objectType="KalturaEventNotificationTemplateFilter">
+                <systemNameEqual>Server_Node_Down</systemNameEqual>
+            </filter>
+            <pager objectType="KalturaFilterPager">
+                <pageSize>1</pageSize>
+            </pager>
+        </request>
+
+        <request service="eventNotificationTemplate" action="update" plugin="eventNotification" partnerId="0">
```

Comment:
Missing the following:
1. The update made to the body of the notification ("is down" => "Status has changed ...")
2. You did not remove from the event conditions the condition the checks the new status is NOT_REGISTERED
3. The system name no longer matches what the notification does so either update the system name (if possible) or create a template.

The same comments should probably be valid for the file without the template.